lib/gui/app/modules/analytics.js

```diff
@@ -50,9 +50,12 @@ resinCorvus.install({
 
 getConfig(configUrl)
   .then((config) => {
+    const probability = _.get(config, [ 'analytics', 'mixpanel', 'probability' ], DEFAULT_PROBABILITY)
+    const mixpanel = _.get(config, [ 'analytics', 'mixpanel' ])
     resinCorvus.setConfigs({
-      mixpanel: getMixpanelConfig(config)
+      mixpanel: isClientEligible(mixpanel, probability)
     })
+    settings.set('sample', probability)
```

Comment:
Can we rename `sample` to something like `mixpanelSample` ?
Also why do we put it in settings?
It could just be local to this file.